runtime/doc/api.txt

```diff
@@ -1707,7 +1707,7 @@ nvim_set_decoration_provider({ns_id}, {opts})
                                specific window. ["win", winid, bufnr, topline,
                                botline_guess]
                              • on_line: called for each buffer line being
-                               redrawn. (The interation with fold lines is
+                               redrawn. (The interaction with fold lines is
```

Comment:
Needs to be fixed here: https://github.com/neovim/neovim/blob/68f61b167e71ca8dd42157b10b3096571c06f39d/src/nvim/api/vim.c#L2950

Reply:
Damn. Good catch!